### ISSUE TITLE
OUJS Fixes

### ISSUE THREAD

**Issue by Martii** [lolamtisch/KissAnimeList#15]
OUJS has made a change recently to require SPDX codes for OSI approved licensing.

In order to improve the appearance of your script homepages it would be appreciated if you could modify your affected script.

Currently this line will be blocked:
https://github.com/lolamtisch/KissAnimeList/blob/168c4197e22643b48a637d56542dbdf19ba9c56d/src/code/01_header.js#L6

In order to continue being posted to OUJS via new scripts and script updates you need to do two things.

1. Pick a Code License from OSI. Typically `MIT` is the least restrictive/protective
2. Use the SPDX short identifier code for CC instead of what you have.

So basically assuming you pick `MIT` that portion of your metadata block should look something like this:

``` js
// @license CC-BY-4.0; https://creativecommons.org/licenses/by/4.0/legalcode
// @license MIT
```

Please note that the Code License of `MIT`, should that be chosen, must be last in the `@license` list... otherwise the server will reject it since the Content License would be last in the list.

Thanks for your patience,
OUJS Staff

**Comment by Martii** [comment]
Thanks... when you get a moment please sync it to OUJS by reimporting it via the import page. I will be going through all the scripts one more time next week and permanently removing those that don't have OSI.

**Comment by lolamtisch** [comment]
All right. I reimported it
